packages/atomic-angular/project.json

```diff
@@ -1,3 +1,32 @@
 {
-  "name": "atomic-angular-builder"
+  "name": "atomic-angular-builder",
+  "targets": {
+    "release:phase1": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "npx -p=@coveo/release bump",
+        "cwd": "packages/atomic-angular/projects/atomic-angular"
+      }
+    },
+    "build-new-version": {
+      "dependsOn": ["release:phase1"],
+      "executor": "nx:run-script",
+      "options": {
+        "script": "build"
+      }
+    },
+    "release:phase2": {
+      "dependsOn": ["release:phase1", "build-new-version"],
+      "executor": "nx:run-commands",
+      "options": {
+        "parallel": false,
+        "commands": [
+          "cp ../../../../../.npmrc .",
+          "npm link ../../../../../node_modules/@coveo/release",
+          "npx -p=@coveo/release npm-publish"
+        ],
```

Comment:
`packages/atomic-angular/projects/atomic-angular/dist` is not a workspace. It's even excluded from the repo by `.gitignore`. Therefore, when we execute a script inside of it, it doesn't inherit from the root `.npmrc` and it doesn't automatically link `@coveo/release`.

Reply:
🤔 why not add it to the ws in the root package.json? (and leave it gitignored)

Reply:
1. It has the same name as the parent package. If I rename the parent, a bunch of stuff would probably break and need to be fixed.
2. `dist` is generated by `npm run build`, and workspaces are included (AFAIK) in `package-lock.json`, therefore the `package-lock.json` would be invalid until somebody runs `npm run build`. Semantically, I don't like this, and I'm worried it could get flaky.